pyiron/lammps/structure.py

```diff
@@ -489,10 +487,11 @@ def structure_full(self):
             id_atom, id_mol, id_species = atom
             # print id_atom, id_mol, id_species
             x, y, z = coords[id_atom]
-            el_id = self._structure.species[id_species].Abbreviation
+            ind = np.argwhere(np.array(species_translate_list) == id_species).flatten()[0]
```

Comment:
Same here, emphasise that only water molecules are supported.

Reply:
This works for all species in the potential config. Not just water molecules